controller/status_test.go

```diff
@@ -6,9 +6,61 @@ import (
 	"reflect"
 	"testing"
 
+	. "github.com/onsi/gomega"
+
 	"github.com/giantswarm/formica/fleet"
 )
 
+func givenSingleUnitStatus(name, sliceID string) fleet.UnitStatus {
```

Comment:
Not sure, I picked it up at TT-S I guess. `given` prefixes more or less helper functions to construct objects in a given state, like a fixture but also for e.g. service objects, not just data objects.